grafana-plugin/src/components/ScheduleCounter/ScheduleCounter.tsx

```diff
@@ -1,14 +1,14 @@
 import React, { FC, useCallback } from 'react';
 
-import { HorizontalGroup, VerticalGroup, Icon, IconButton, Tooltip } from '@grafana/ui';
+import { HorizontalGroup, VerticalGroup, Icon, IconButton, Tooltip, IconName } from '@grafana/ui';
 import cn from 'classnames/bind';
 
-import Text from 'components/Text/Text';
+import Text, { TextType } from 'components/Text/Text';
 
 import styles from './ScheduleCounter.module.css';
 
 interface ScheduleCounterProps {
-  type: 'link' | 'warning';
+  type: Partial<TextType>;
```

Comment:
`Partial` should be used for types not for unions, `TextType` here is a string union type.

Reply:
what is better in this case?

Reply:
Just leave `'link' | 'warning'`, there's also `Extract<TextType, 'warning' | 'danger'>` if you prefer this version